server/server.go

```diff
@@ -0,0 +1,168 @@
+package server
+
+import (
+	"fmt"
+	"net"
+	"context"
+	"strings"
+	"romano.com/proto"
+	"romano.com/mongo"
+	"romano.com/model/fileModel"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Server struct {
+	port string
+}
+
+const (
+	mongoID string = "_id";
+)
+
+var mongoManager mongo.Controller;
+
+func New(port string) Server {
```

Comment:
Should return `*Server` to match the struct's methods, which are defined on `*Server` and not `Server`.